### PR TITLE
Link Aliucord to Discord

### LINKED ISSUE #334: Request: link Discord icon to Aliucord (com.aliucord)
Please link Discord's icon to Aliucord (a Discord modification).
Thanks!

### PR DESCRIPTION
**Pull request by Randomblock1**
Fixes #334 
Just links Aliucord to the Discord icon.
Note: Both entries are named Discord so they won't appear as duplicates in the app. I don't know if this is the correct way to do it, but it's the way I did it.

### CODE REVIEW

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -81,6 +81,7 @@
     <icon drawable="@drawable/dialer" package="com.truecaller" name="Phone" />
     <icon drawable="@drawable/digital_wellbeing" package="com.google.android.apps.wellbeing" name="Digital Wellbeing" />
     <icon drawable="@drawable/discord" package="com.discord" name="Discord" />
+    <icon drawable="@drawable/discord" package="com.aliucord" name="Discord" />
```
Comment (patrykmichalik) [inline app/src/main/res/xml/grayscale_icon_map.xml:84]:
```suggestion
    <icon drawable="@drawable/discord" package="com.aliucord" name="Aliucord" />
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
I’d suggest changing the value of the `name` attribute to `"Aliucord"`. This is so that the right icon is shown after you type “Aliucord” into the search bar. I’ll most likely add some code that filters out duplicate icons from the home screen based on the value of the `drawable` attribute.

**Comment by Randomblock1** [comment]
OK, fixed it.

**Comment by patrykmichalik** [comment]
Thanks, @Randomblock1!

**Pull request merged**